js/src/services/vue.service.js

```diff
@@ -15,6 +16,7 @@ let atkComponents = {
   'atk-item-search' : itemSearch,
   'atk-multiline'   : multiLine,
   'atk-tree-item-selector' : treeItemSelector,
+  'atk-query-builder' : VueQueryBuilder,
```

Comment:
@ibelar why is this needed? Do we need to change js for every Vue based component added?